content/overview/pricing/rates.md

```diff
@@ -72,11 +72,11 @@ Free of charge and without any paperwork, anyone with a government email address
 
 ## Quota pricing
 
-Quotas are priced at $0.0033 per MB reserved per day and assessed monthly. In practice, this works out to $99 per GB per month, based on a 30-day month:
+Quotas are priced at $0.0033 per MB reserved per day and assessed monthly. In practice, this works out to about $100 per GB per month:
 
-`$0.0033 * 1000 MB * 30 days = $99 per GB per month`
+`$0.0033 * 1024 MB * 30 days = $101.37 per GB per month`
 
-The monthly cost will vary slightly from $92.40 per GB in a 28-day month to $102.30 per GB in a 31-day month.
+The monthly cost will vary slightly from $94.62 per GB in a 28-day month to $104.76 per GB in a 31-day month.
```

Comment:
Do we really need this clarification or detail here? I think it adds more text to a page that's already wordy.